app/src/main/java/com/sun_asterisk/moviedb_49/data/model/Genres.kt

```diff
@@ -1,6 +1,10 @@
 package com.sun_asterisk.moviedb_49.data.model
 
 data class Genres(
-    var id: Int,
-    var name: String
-)
+    var id: Int?,
+    var name: String?
+) {
+    override fun toString(): String {
```

Comment:
override method này ra để làm gì vậy e?

Reply:
dạ để em in ra name list genres trong list<Movie> á anh. Vì em đổ dâta của movie lên màng hình mà mình cần lấy list name của genres